src/main/java/org/broadinstitute/hellbender/tools/exome/ACSModeller.java

```diff
@@ -0,0 +1,150 @@
+package org.broadinstitute.hellbender.tools.exome;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.broadinstitute.hellbender.tools.exome.allelefraction.AlleleFractionModeller;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.mcmc.PosteriorSummary;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represents an ACS segmented model for copy ratio and allele fraction.
+ *
+ * @author Samuel Lee &lt;slee@broadinstitute.org&gt;
+ */
+public final class ACSModeller {
+    protected static final String INITIAL_SEG_FILE_TAG = "sim-0";
```

Comment:
good point, i thought about this issue but didn't come up with a good scheme.  the natural order is:

1) seg file from RCS
2) MAF SNP segments (-MAF, currently)
3) segment union (-union)
4) after small-segment merging (-no-small)
5) initial fit (-sim-0)
6) fits after each iteration of similar-segment merging (-sim-#)
7) final fit (-sim-final)